README.md

```diff
@@ -60,70 +68,70 @@ sudo sh -c 'curl -L https://github.com/redhat-developer/odo/releases/download/v0
 </details>
 
 <details>
-<summary> :package: :checkered_flag: Windows</summary>
-
-#### In order to correctly use odo you must add it to your PATH environment variable
+<summary> Windows</summary>
 
 1. Download the `odo-windows-amd64.exe.gz` file from the [GitHub releases page](https://github.com/redhat-developer/odo/releases).
-2. Extract the file
+2. Extract the file.
 3. Add the location of extracted binary to your PATH environment variable by following [this Wiki page](https://github.com/redhat-developer/odo/wiki/Setting-PATH-variable-on-Windows).
 
 </details>
 
-#### For a list of other methods such as installing the latest mastery binary, or specific OS installations, visit the [installation page](/docs/installation.md).
-
-## Purpose
+For a list of other methods such as installing the latest binary on specific OS installations, visit the [installation page](/docs/installation.md).
 
-OpenShift Do (odo) is a CLI tool for developers who are writing, building, and deploying applications on OpenShift. With odo, developers get an opinionated CLI tool that supports fast, iterative development which abstracts away Kubernetes and OpenShift concepts, thus allowing them to focus on what's most important to them: code.
```

Comment:
I don't like how we move this section back above. Makes the entire front section very cluttered...

Compare both side-by-side and you'll see what I mean.

Reply:
We need an introduction explaining why a user would want to use odo before directing him to the installation section. The key features as they were earlier were brief and imho did not give a holistic idea about odo. 
Purpose of odo/what odo does for the user should compel the user to install it. Having it after install section adds no value.
I agree, with the wall of text comment, I have streamlined the section, ptal and let me know if this is ok.